infrastructure/ansible/roles/certbot/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+- name: install software properties common
+  become: true
+  apt:
+    state: present
+    name: software-properties-common
+  
+- name: install certbot apt-repositories
+  become: true
+  apt_repository:
+    state: present
+    repo: "{{ item }}"
+  loop:
+    - "deb http://archive.ubuntu.com/ubuntu/ bionic universe"
+    - "deb http://archive.ubuntu.com/ubuntu/ bionic-updates universe"
+    - "deb http://security.ubuntu.com/ubuntu/ bionic-security universe"
+    - "ppa:certbot/certbot"
+
+- name: install certbot and python for certbot on nginx
+  become: true
+  apt:
+    update_cache: yes
+    name:
+      - certbot
+      - python-certbot-nginx
+
+- name: look for certbot generated key file
+  become: true
+  stat:
+    path: /etc/letsencrypt/live/prerelease.triplea-game.org/cert.pem
+  register: has_certbot_pem
+
+- name: run certbot
+  become: true
+  command: "certbot --nginx -n -m tripleabuilderbot@gmail.com --agree-tos -d {{ inventory_hostname }}"
```

Comment:
We don't have any mail forwarding for this email don't we?
I just got (2 days ago) an email from the certbot utility telling me that it failed to renew the MARTI HTTP certificates.
Turns out that there was (and still is, but I found a workaround) a DNSSEC issue with namecheap that fails for a nonexistent subdomain, which ultimately caused the check to fail.
Without this email we wouldn't have noticed until it's too late, so I'm convinced there's definitely some value in using an actual email here.

Reply:
Oh and another thing: By adding `--rsa-key-size 4096` we get a stronger RSA key.
I use this for every other server we use certbot on.

Reply:
tripleabuilderbot@gmail.com  is a real email. If we have a dev-forwaring list, that would be good to use. I just sent a test email to: tripleadevelopers@gmail.com and got nothing - is that a working distro? Arguably we should also encode the email as well so that it does not get scraped and yield spam.

Added the key-size flag: 69a240eed; The command here is based on what we previously had in the README